README.md

```diff
@@ -77,22 +78,23 @@ yarn add nodejieba
+| indexPages                       | boolean              | `false`   | Whether to index pages.                                                                                                                      |
+| docsRouteBasePath                | string \| string[]   | `"/docs"` | Base route path(s) of docs. Slash at beginning is not required.                                                                              |
+| blogRouteBasePath                | string \| string[]   | `"/blog"` | Base route path(s) of blog. Slash at beginning is not required.                                                                              |
+| language                         | string \| string[]   | `"en"`    | All [lunr-languages](https://github.com/MihaiValentin/lunr-languages) supported languages, + `zh` 🔥.                                        |
+| hashed                           | boolean              | `false`   | Whether to add a hashed query when fetching index (based on the content hash of all indexed `*.md` in `docsDir` and `blogDir` if applicable) |
+| docsDir                          | string \| string[]   | `"docs"`  | The dir(s) of docs to get the content hash, it's relative to the dir of your project.                                                        |
+| blogDir                          | string \| string[]   | `"blog"`  | Just like the `docsDir` but applied to blog.                                                                                                 |
+| removeDefaultStopWordFilter      | boolean              | `false`   | Sometimes people (E.g., us) want to keep the English stop words as indexed, since they maybe are relevant in programming docs.               |
+| highlightSearchTermsOnTargetPage | boolean              | `false`   | Highlight search terms on target page.                                                                                                       |
+| searchResultLimits               | number               | `8`       | Limit the search results.                                                                                                                    |
+| searchResultContextMaxLength     | number               | `50`      | Set the max length of characters of each search result to show.                                                                              |
+| translations                     | TranslationMap       | -         | Set translations of this plugin, see [docs below](#translations).                                                                            |
+| i18n                             | TranslationLocaleMap | -         | Allow to translate the plugin for more than one locale, see [docs below](#translations).                                                     |
```

Comment:
I'm not familiar with i18n in Docusaurus right now, but I think we should follow [the official i18n guide of Docusaurus](https://docusaurus.io/docs/i18n/introduction), it seems that it supports i18n of plugins which is defined by files in specific path.

By the way, I have a limited time to maintain this project and to learn new features from Docusaurus, so it may take days to review a PR.